lib/core/src/lib/card-view/components/card-view-arrayitem/card-view-arrayitem.component.scss

```diff
@@ -40,6 +40,10 @@
         display: flex;
         place-content: center space-between;
         align-items: center;
+        background: var(--adf-card-view-array-background);
```

Comment:
I would name it --adf-card-view-array**-item**-background, but it is up to you

Reply:
Done